docs/docs/developers/versions-updating.md

```diff
@@ -73,42 +72,43 @@ To set `VERSION` for a particular git tag, eg for [aztec-package-v**0.35.0**](ht
 VERSION=0.35.0 aztec-up
 ```
 
-2. Updating aztec-nr and individual @aztec dependencies:
+2. Update aztec-nr and individual @aztec dependencies:
 
-Inside your project run:
+<!-- Inside your project run:
```

Comment:
So the `update` command is gone? This has to be done manually now? :slightly_frowning_face:

Reply:
it was added back :)